extension/chrome/settings/index.ts

```diff
@@ -309,6 +311,38 @@ Catch.try(async () => {
     }));
   };
 
+  const renderBlogPostList = async () => {
+    const errorRetrievingPosts = () => {
+      Xss.sanitizeAppend('.blog_post_list', `<div class="line>Error retrieving blog posts...try again later.</div>\n`);
+    };
+    try {
```

Comment:
You won't need the try/catch here either because we will be handling it one step up with the `.catch()`